keel-deliveryconfig-plugin/keel-deliveryconfig-plugin.gradle

```diff
@@ -7,6 +7,10 @@ dependencies {
   implementation project(":keel-plugin")
   implementation project(":keel-front50")
 
+  implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion"
```

Comment:
much boiler

very plate

Not sure there is a better option really though without reinventing `spinnaker.group('kotlinySpringyCommonDependencies')`

Reply:
Are the kotlin spring dependencies significantly different than the normal ones? I'd be delighted with a group if there are more than a couple.

Reply:
I tend to never use the groups because I'm fussy about what I import (as you can tell from the fact you're fixing my shitty dependency tangle in this PR) and I have to go refer to spinnaker/spinnaker-dependencies in order to understand what they do. If we want to group them we could do an `apply(from: "$rootDir/gradle/kotling-spring-common.gradle")` or something.